File: 프로그래머스/unrated/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    for x in range(len(arr)):
        if (arr[x] >= 50) and (arr[x] % 2 == 0):
            arr[x] = int(arr[x] / 2)
        elif (arr[x] < 50) and (arr[x] % 2 == 1):
            arr[x] = arr[x] * 2
    return arr

File: 프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.java
class Solution {
    public int[] solution(int[] arr) {
        int[] answer = new int[arr.length];
        
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] >= 50 && arr[i] % 2 == 0) {
                answer[i] = arr[i] / 2;
            } else if (50 > arr[i] && arr[i] % 2 == 1) {
                answer[i] = arr[i] * 2;
            } else {
                answer[i] = arr[i];
            }
        }
        return answer;
    }
}